misc/minetest.exe.manifest

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+    <assemblyIdentity type="win32" name="minetest" version="0.0.0.0" />
```

Comment:
Why version 0?

Reply:
I don't want to add this file to the version bump scripts etc. so this is just a placeholder.
Doesn't show in the "Properties" tab of explorer either.